sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/Uuid.java

```diff
@@ -49,6 +50,9 @@ public static Uuid random() {
     } catch (IOException e) {
       throw new RuntimeException("Should never have an IOException since there is no io.", e);
     }
-    return Uuid.of(output.toByteString());
+    // Encode to Base64 so the random UUIDs are valid if consumed from the Cloud Pub/Sub client.
+    return Uuid.of(
+        ByteString.copyFrom(
+            Base64.getEncoder().encode(output.toByteString().asReadOnlyByteBuffer())));
```

Comment:
This change comes from https://github.com/googleapis/java-pubsublite/pull/552/files

CC: @dpcollins-google